CoreDataPlatform/Entities/CDAddress+CoreDataProperties.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  CDAddress+CoreDataProperties.swift
+//  CleanArchitectureRxSwift
+//
+//  Created by Andrey Yastrebov on 10.03.17.
+//  Copyright © 2017 sergdort. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+
+extension CDAddress {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<CDAddress> {
+        return NSFetchRequest<CDAddress>(entityName: "CDAddress");
+    }
+
+    @NSManaged public var city: String?
```

Comment:
I think you can safely remove these optionals as long as field is not optional in core data
<img width="255" alt="screen shot 2017-04-02 at 16 06 27" src="https://cloud.githubusercontent.com/assets/4622322/24588347/5fce2eb2-17be-11e7-89c2-4684204d0599.png">